games/snake.py

```diff
@@ -62,13 +66,13 @@ def move_snake(self, key):
             new_head[1] -= 1
         if key in [curses.KEY_RIGHT, ord('l')]:
             new_head[1] += 1
-        self.snake.insert(0, new_head)
+            self.snake.insert(0, new_head)
```

Comment:
This should apply globally to the move_snake method. The extra indent make this only apparent to the last if statement